ctx.go

```diff
@@ -508,6 +508,47 @@ func (c *Ctx) SetCipherList(list string) error {
 	return nil
 }
 
+// SetCipherSuites sets the list of available ciphers used for tls1.3.
+func (c *Ctx) SetCipherSuites(list string) error {
+	runtime.LockOSThread()
+	defer runtime.UnlockOSThread()
+	clist := C.CString(list)
+	defer C.free(unsafe.Pointer(clist))
+	if int(C.SSL_CTX_set_ciphersuites(c.ctx, clist)) != 1 {
+		return errorFromErrorQueue()
+	}
+	return nil
+}
+
+// SetGroupsList
+func (c *Ctx) SetGroupsList(list string) error {
+	runtime.LockOSThread()
+	defer runtime.UnlockOSThread()
+	clist := C.CString(list)
+	defer C.free(unsafe.Pointer(clist))
+	// # define SSL_CTX_set1_groups_list(ctx, s) \
+	//     SSL_CTX_ctrl(ctx,SSL_CTRL_SET_GROUPS_LIST,0,(char *)(s))
+	// # define SSL_CTRL_SET_GROUPS_LIST                92
+	if int(C.SSL_CTX_ctrl(c.ctx, C.int(92), C.long(0),
```

Comment:
For macros, the general style in the codebase is to define an `X_$macro` function in `shim.c/h` and then call that.
See `SSL_CTX_set1_curves` for an example https://github.com/pexip/go-openssl/blob/07cafbe4ec06403a1cd2eac33172389841649f0e/shim.c#L184-L186
I think it would be worthwhile doing that here as then the go code matches the call syntax of the openssl man page for the macro.